<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodingStandardPage</class>
 <widget class="QWidget" name="CodingStandardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>Coding Standard Generation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="initialChoicePage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="instructionLabel">
         <property name="text">
          <string>Choose how to create the Coding Standard document(s) for the detected technologies.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="techListWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="aiProposedButton">
           <property name="text">
            <string>Generate with AI Proposal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pmGuidedButton">
           <property name="text">
            <string>Generate with PM Guidelines</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pmDefinePage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="pmDefineHeaderLabel">
         <property name="text">
          <string>Define Guidelines for [Technology]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="pmInputTabWidget">
         <widget class="QWidget" name="textInputTab">
          <attribute name="title">
           <string>Enter Text Guidelines</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_text">
           <item>
            <widget class="QPlainTextEdit" name="pmGuidelinesTextEdit"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="uploadTab">
          <attribute name="title">
           <string>Upload Guideline Documents</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_upload">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_upload">
             <item>
              <widget class="QLineEdit" name="uploadPathLineEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="browseFilesButton">
               <property name="text">
                <string>Browse...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_upload">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="cancelButton_1">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="generateFromGuidelinesButton">
           <property name="text">
            <string>Analyze Guidelines &amp;&amp; Generate Draft</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="reviewPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="reviewHeaderLabel">
         <property name="text">
          <string>Review Coding Standard for [Technology]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="reviewTabWidget">
         <widget class="QWidget" name="specDraftTab">
          <attribute name="title">
           <string>Standard Draft</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QTextEdit" name="standardTextEdit"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="aiAnalysisTab">
          <attribute name="title">
           <string>AI Analysis &amp;&amp; Suggestions</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QTextEdit" name="aiAnalysisTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="feedbackTab">
          <attribute name="title">
           <string>Your Feedback</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QLabel" name="feedbackLabel">
             <property name="text">
              <string>Provide feedback and clarifications below to have the AI generate a new version.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="feedbackTextEdit"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="cancelButton_2">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="refineButton">
           <property name="text">
            <string>Submit Feedback &amp;&amp; Refine</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="approveButton">
           <property name="text">
            <string>Approve Standard</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="processingPage">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="processingLabel">
         <property name="text">
          <string>Processing... Please wait.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>